cudaq-1.0.0/cudaq.egg-info/requires.txt

```diff
@@ -0,0 +1 @@
+cudaq-cu11==1.0.0
```

Comment:
Why is this hardcoded to be cu11? 
A lot of things that are checked in here would/should be autogenerated by the wheel build. How are you building the cudaq source distribution (what you call the cudaq-1.0.0)?